src/components/banner/AuthenticatedUserView.tsx

```diff
@@ -9,51 +8,102 @@ import MenuItem from "@mui/material/MenuItem";
 import Typography from "@mui/material/Typography";
 import { User } from "next-auth";
 import Link from "next/link";
-import { useState } from "react";
+import { usePathname } from "next/navigation";
```

Comment:
@ChrisLiva - this btw is one of the things I find slightly annoying about import organization. This diff looks more complicated that it strictly is, but since diffs are completely text-based, it doesn't really know that here.